Civi/Core/SqlTriggers.php

```diff
@@ -120,7 +120,7 @@ public function createTriggers($info, $onlyTableName = NULL) {
           );
           $variables = str_replace($template_params,
             $template_values,
-            \CRM_Utils_Array::value('variables', $value)
+            \CRM_Utils_Array::value('variables', $value, '')
```

Comment:
or just `$value ?? ''` & remove `\CRM_Utils_Array::value(`

Reply:
ok updated to that style now @eileenmcnaughton